java/aiservices/openai/src/main/java/com/microsoft/semantickernel/aiservices/openai/chatcompletion/OpenAIChatCompletion.java

```diff
@@ -198,17 +198,21 @@ private Mono<List<ChatMessageContent<?>>> internalChatMessageContentsAsync(
                                                     functionResult.getResult(),
                                                     functionToolCall.getId());
 
-                                                msgs.add(requestToolMessage);
-                                                return msgs;
+                                                ArrayList<ChatRequestMessage> res = new ArrayList<>(
```

Comment:
Since you added this line, I wonder if you can remove the need of creating a new ArrayList here: https://github.com/microsoft/semantic-kernel/blob/c3d4fea0ca1716c91d6c64e2e10612872ce1612e/java/aiservices/openai/src/main/java/com/microsoft/semantickernel/aiservices/openai/chatcompletion/OpenAIChatCompletion.java#L119